packages/peregrine/lib/talons/CheckoutPage/CheckoutError.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Other checkout place order mutation errors can include
+ * the following list:
+ *
+ * https://devdocs.magento.com/guides/v2.3/graphql/mutations/place-order.html#errors
+ */
+
+const paymentErrorMessage =
```

Comment:
Maybe leave a comment explaining that this a heuristic, rather than something to be rendered?

Reply:
This may be the correct use case for an enum containing all of the constants from the [documentation linked elsewhere](https://devdocs.magento.com/guides/v2.3/graphql/mutations/place-order.html#errors). Perhaps import this one?